### PR TITLE
Remove unused paket files

### PR DESCRIPTION
**Pull request by flavios98**

### CODE REVIEW

**Review by janhohenheim — CHANGES_REQUESTED** [review]
- `Scripts/install_paket.sh` neeeds to be added to `.travis.yml`.
- Since we don't have a Windows CI at the moment, you can remove `Scripts/InstallPaket.ps1`
- Packages should have no lock file. Determining the exact library versions used should be up to the consumer.

File: .gitignore [hunk 1]
```diff
@@ -242,7 +242,6 @@ _Pvt_Extensions
 
 # Paket dependency manager
 paket-files/
-paket.lock
 paket.bootstrapper.exe
```
Comment (janhohenheim) [inline .gitignore]: This line can be removed as well

**Review by janhohenheim — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (flavios98) [reply .gitignore]: 92fdaa3bae09500a265c107c909a346edd8e2fca

**Comment by flavios98** [comment]
92fdaa3bae09500a265c107c909a346edd8e2fca

**Review by janhohenheim — APPROVED** [review]

**Pull request merged**